CRM/Contribute/Form/Contribution.php

```diff
@@ -2541,4 +2530,26 @@ private function buildRecur(): void {
     $form->add('checkbox', 'is_recur', $is_recur_label, NULL);
   }
 
+  /**
+   *
+   * @return bool|null
+   * @throws \CRM_Core_Exception
+   */
+  private function isPayLater(): bool {
```

Comment:
This function doesn't seem to get called so it still comes up incomplete transaction. It does get called for the credit card form, but pay later makes no sense there.